src/lib/cimb_translator/Config.h

```diff
@@ -1,30 +1,105 @@
 /* This code is subject to the terms of the Mozilla Public License, v.2.0. http://mozilla.org/MPL/2.0/. */
 #pragma once
 
-// probably defaults with a pimpl to read/cache if there's a (yaml?) config locally?
+#include "GridConf.h"
+
 namespace cimbar
 {
-	namespace Config
+	class Config
 	{
-		bool dark();
-
-		unsigned color_bits();
-		unsigned symbol_bits();
-		unsigned bits_per_cell();
-		unsigned ecc_bytes();
-
-		int image_size();
-		unsigned anchor_size();
-
-		unsigned cell_size();
-		unsigned cell_spacing();
-		unsigned num_cells();
-		unsigned corner_padding();
-		unsigned interleave_blocks();
-		unsigned interleave_partitions();
-
-		unsigned fountain_chunk_size(unsigned ecc, unsigned bitspercell=0);
-		unsigned fountain_chunks_per_frame();
-		unsigned compression_level();
-	}
+	protected:
+		using GridConf = Conf8x8;
```

Comment:
Where one would swap in an alternate grid config, if they wanted to do such a thing.